### PR TITLE
EZPZ_CU-867758mbf: FAQs Schema & Resolvers

### PR DESCRIPTION
**Pull request by arfusop**

### CODE REVIEW

**Comment by arfusop** [comment]
Task linked: [CU-867758mbf Create the FAQ Schema & typeDefs](https://app.clickup.com/t/867758mbf)

File: gql/typeDefs.js [hunk 1]
```diff
@@ -78,7 +80,7 @@ const typeDefs = gql`
     }
 
     union UserResponse = UserSuccess | Errors
-    union MultiUserResponse = MultipleUsersSuccess | Errors
+    union MultiUserResponse = MultiUsersSuccess | Errors
```
Comment (KevinLaCarrubba) [inline gql/typeDefs.js:83]: it just bothers me that MultiUserResponse doesnt have a S after User but MultiUsersSuccess has the S on Users

Comment (KevinLaCarrubba) [reply gql/typeDefs.js:83]: i feel like i like it more without the S  but idk

**Review by KevinLaCarrubba — APPROVED** [review]

File: gql/typeDefs.js (hunk 1 above)
Comment (arfusop) [reply gql/typeDefs.js:83]: I'll address this in the next branch that redoes the CMS. Good thinking

**Pull request merged**